bindings/python/crocoddyl/multibody/contact-base.cpp

```diff
@@ -37,6 +37,7 @@ void exposeContactAbstract() {
            "Compute the derivatives of contact holonomic constraint.\n\n"
            "The rigid contact model throught acceleration-base holonomic constraint\n"
            "of the contact frame placement.\n"
+           "It assumes that that calc has been run first.\n"
```

Comment:
You should change "that that" by that :)

I think you can simply run the following command in your crocoddyl folder to change all occurrence of "that that" by "that":

```
find ./ -type f -readable -writable -exec sed -i "s/that that/that/g" {} \;
```

Reply:
Thanks for the command.
Now it is fixed!